src/style.css

```diff
@@ -1 +1,118 @@
-/* empty source */
+/* empty source */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+  color: var(--text-secondary-color);
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__product-image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--text-accent-color);
+}
+
+.card__product-code {
+  margin: 0 0 16px;
+}
+
+.stars__review {
+  color: var(--text-accent-color);
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 24px;
+  align-items: center;
+}
+
+.stars__wraper {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__wraper_star--1 .stars__star:nth-child(-n + 1),
+.stars__wraper_star--2 .stars__star:nth-child(-n + 2),
+.stars__wraper_star--3 .stars__star:nth-child(-n + 3),
+.stars__wraper_star--4 .stars__star:nth-child(-n + 4),
+.stars__wraper_star--5 .stars__star:nth-child(-n + 5) {
+  background-image: url("./images/star-active.svg");
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 16px;
+}
+
+.price__title {
+  line-height: 18px;
+  font-size: 12px;
+}
+
+.price__Amount {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--text-accent-color);
+}
```

Comment:
If you think `price` is a seaprate block that can be used outside a card 
then OK
But if not, just make it and element of a card